assignments/callbacks.js

```diff
@@ -21,28 +21,47 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, function(length) {
+  console.log(length);
+});
 
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  return cb(arr.slice(-1)[0]);
```

Comment:
compare to solution code:

function last(arr, cb) {
  cb(arr[arr.length - 1]);
}